src/main/java/Duke.java

```diff
@@ -1,10 +1,89 @@
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.Scanner;
+
 public class Duke {
```

Comment:
the websites says: "Class variables should never be declared public"
So, perhaps it should not public.